flow-typed/npm/classnames_v2.x.x.js

```diff
@@ -0,0 +1,16 @@
+// flow-typed signature: cf6332fcf9a3398cffb131f7da90662b
+// flow-typed version: dc0ded3d57/classnames_v2.x.x/flow_>=v0.28.x
```

Comment:
Offline answer: something that should be discussed as well. here we are kind of "pinning" to a specific version of this typing (shielding from possible typing breakages on flow typed). Do we consider this relevant? Should we? to be discussed